getting-started/kubernetes/managed-public-cloud/eks.md

```diff
@@ -11,13 +11,53 @@ Enable Calico in EKS managed Kubernetes service.
 
 EKS has built-in support for Calico, providing a robust implementation of the full Kubernetes Network Policy API. EKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the Calico Network Policy API.
 
+You can also use Calico for networking on EKS in place of the default AWS VPC networking. This allows you to take advantage of the full set of Calico networking features, including Calico's flexible IP address managemenet capabilities.
```

Comment:
I wonder if we should say something about without the need to use IP addresses from the underlying VPC?